web_app/tests/test_user.py

```diff
@@ -21,26 +19,30 @@ async def async_client():
         ),
     ],
 )
-async def test_get_user_contract(
-    async_client, wallet_id: str, expected_contract_address: str
-):
+async def test_get_user_contract(wallet_id: str, expected_contract_address: str):
     """
-    Test get_user_contract endpoint,
-    where wallet_id = "0x27994c503bd8c32525fbdaf9d398bdd4e86757988c64581b055a06c5955ea49" or ""
-    :param async_client: httpx.AsyncClient
+    Test get_user_contract endpoint
+    :param wallet_id: "" or "0x27994c503bd8c32525fbdaf9d398bdd4e86757988c64581b055a06c5955ea49"
+    :param expected_contract_address: "" or "0x698b63df00be56ba39447c9b9ca576ffd0edba0526d98b3e8e4a902ffcf12f0"
     :return: None
     """
-    response = await async_client.get(
+    response = client.get(
         url="/api/get-user-contract",
         params={
             "wallet_id": wallet_id,
         },
     )
     response_json = response.json()
 
-    assert response.is_success
-    assert isinstance(response_json, str)
-    assert response_json == expected_contract_address or response_json == ""
+    if response.is_success:
+        assert response.is_success
+        assert isinstance(response_json, str)
+        assert response_json == expected_contract_address
+    else:
+        assert not response.is_success
+        assert isinstance(response_json, dict)
+        assert "detail" in response_json
```

Comment:
please, make it shorter:
`response_json.get("detail")` in ("User not found", "Contract not deployed")